### ISSUE TITLE
osTicket Version	v1.10-rc.3 (907ec36) - Due Date in Task not showing

### ISSUE THREAD

**Issue by ramkinavy** [osTicket/osTicket#3429]
Creating tasks with due dates, but it is blank when the task is open. even when the tasks are sorted by due date, the due date column is blank. please find attached snapshots

<img width="726" alt="screen shot 2016-10-27 at 10 04 36 pm" src="https://cloud.githubusercontent.com/assets/8140237/19776460/c73aa736-9c91-11e6-862e-a71cff31cc42.png">
<img width="488" alt="screen shot 2016-10-27 at 10 04 58 pm" src="https://cloud.githubusercontent.com/assets/8140237/19776466/ce6aeeee-9c91-11e6-9804-275248facd4a.png">
<img width="960" alt="screen shot 2016-10-27 at 10 06 52 pm" src="https://cloud.githubusercontent.com/assets/8140237/19776474/d47f0afe-9c91-11e6-96ad-7b00af46fc1a.png">

**Comment by ntozier** [comment]
Did you search the existing issues?
This sounds like it might be a duplicate of: https://github.com/osTicket/osTicket/issues/2799

**Comment by ramkinavy** [comment]
Thanks for the update 

Get Outlook for iOS




On Thu, Oct 27, 2016 at 10:14 PM +0530, "ntozier" <notifications@github.com> wrote:












Did you search the existing issues?

This sounds like it might be a duplicate of: #2799

**Comment by JediKev** [comment]
I have a Pull Request (#3430) pending on this issue.

**Comment by rayfoss** [comment]
duplicate of 2799
